dev/scalastyle

```diff
@@ -29,6 +29,7 @@ ERRORS=$(echo -e "q\n" \
         -Phive-thriftserver \
         -Pspark-ganglia-lgpl \
         -Pdocker-integration-tests \
+	-Pkubernetes-integration-tests \
```

Comment:
I recall the way this is setup right now, buildin the profile will try to execute the tests?

Reply:
I mean we aren't building the profile per-se, we're only evaluating `scalastyle` `test:scalastyle` on it, but let me double check locally.

Reply:
Running `./dev/scalastyle` after this change locally does not trigger the k8s integration tests